client/src/components/App.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import AddTrack from './AddTrack.jsx';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      songs: [],
+    };
+    this.fetchRelatedSongs = this.fetchRelatedSongs.bind(this);
+    this.abbreviateNumber = this.abbreviateNumber.bind(this);
+  }
+
+  componentDidMount() {
+    this.fetchRelatedSongs()
+      .then((songs) => {
+        this.setState({ songs });
+      });
+  }
+
+  fetchRelatedSongs() {
+    return fetch("/api/songs/:songid/related", {
```

Comment:
You can probably join these two api calls into 1. You can grab the related track list and send back the songs table data from that same API call